src/components/Popover/PopoverContent.module.css

```
@@ -0,0 +1,21 @@
+.popover-content {
+  --popover_content-box_shadow: 1px 1px 4px rgba(0, 0, 0, 0.25);
+}
+
+.popover-content--yellow {
+  --popover_content-color-background-yellow: #fbf6bd;
+  background-color: var(--popover_content-color-background-yellow);
+  box-shadow: var(--popover_content-box_shadow);
+}
+
+.popover-content--red {
+  --popover_content-color-background-red: #ffc1d2;
+  background-color: var(--popover_content-color-background-red);
+  box-shadow: var(--popover_content-box_shadow);
+}
+
+.popover-content--white {
+  --popover_content-color-background-white: #ffffff;
+  background-color: var(--popover_content-color-background-white);
+  box-shadow: var(--popover_content-box_shadow);
+}
```

Same here as other css - move common parts to base, and add overrides to modifiers:

```suggestion
.popover-content {
  background-color: var(--popover_content-color-background);
  box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.25);
}

.popover-content--yellow {
  --popover_content-color-background: #fbf6bd;
}

.popover-content--red {
  --popover_content-color-background: #ffc1d2;
}

.popover-content--white {
  --popover_content-color-background: #ffffff;
}
```